### PR TITLE
Fix schedule annotation and some aop issue

### LINKED ISSUE #643: Spring boot starter disables scheduling
Hi,
i use the springboot starter of javameldoy for my springboot application. At the switch from version 1.67.0 to 1.68.0 some things go wrong for spring scheduling. I use the springboot annotation `@EnableScheduling` so that the springframework can search for `@Scheduling` annotated beans.

In 1.67.0 everything works as expected but since version 1.68.0 of javamelody my scheduling classes are no longer found by the `ScheduledAnnotationBeanPostProcessor`. 
In general it seems like every bean with scheduling annotations can no longer be resolved by spring with java melody 1.68.0

### PR DESCRIPTION
**Pull request by jfbreault**
fix issue #643

### CODE REVIEW

File: javamelody-spring-boot-starter/src/main/java/net/bull/javamelody/JavaMelodyConfigurationProperties.java [hunk 1]
```diff
@@ -39,6 +39,7 @@
 	private boolean enabled = true;
 	private String excludedDatasources;
 	private Map<String, String> initParameters = new HashMap<String, String>();
+	private boolean springMonitoringEnabled = true;
```
Comment (jfbreault) [inline javamelody-spring-boot-starter/src/main/java/net/bull/javamelody/JavaMelodyConfigurationProperties.java]: Added to generate meta-data for properties/yml autocompletition in ide

File: javamelody-spring-boot-starter/src/test/java/net/bull/javamelody/TestJavaMelodyAutoConfiguration.java [hunk 2]
```diff
@@ -59,25 +59,23 @@
...
 
-	/**
-	 * test.
-	 */
-	@SuppressWarnings("unchecked")
 	@Test
-	public void testJavaMelodyAutoConfigurationIsCreated() {
-		// It should create a session listener.
-		final ServletListenerRegistrationBean<EventListener> sessionListenerRegistrationBean = context
+	public void testJavaMelodyAutoConfigurationIsCreatedThenItShouldCreateASessionListener() {
```
Comment (jfbreault) [inline javamelody-spring-boot-starter/src/test/java/net/bull/javamelody/TestJavaMelodyAutoConfiguration.java]: I could revert it but a test that test about 50 functionality is bad. I tried to split it according to comment, if comment are necessary to know what we want to test, we should split it. Personally, I would have split it more.

File: javamelody-spring-boot-starter/src/main/java/net/bull/javamelody/JavaMelodyAutoConfiguration.java [hunk 3]
```diff
@@ -143,15 +142,6 @@ public FilterRegistrationBean monitoringFilter(JavaMelodyConfigurationProperties
 	}
 
 	/**
-	 * @return DefaultAdvisorAutoProxyCreator
-	 */
-	@Bean
-	@ConditionalOnMissingBean(DefaultAdvisorAutoProxyCreator.class)
-	public DefaultAdvisorAutoProxyCreator defaultAdvisorAutoProxyCreator() {
```
Comment (jfbreault) [inline javamelody-spring-boot-starter/src/main/java/net/bull/javamelody/JavaMelodyAutoConfiguration.java]: This is the cause of the proxy taget class not working correctly and break `@Scheduled` annotion

**Comment by evernat** [comment]
issue is fixed by ddff401

**Pull request closed without merging**